File: simulator/atan.ml
let a = 10.0 *. atan 0.0 in (* 0 *)
let b = 10.0 *. atan 1.0 in (* 8 *)
let c = 10.0 *. atan -1.0 in (* -8 *)
let d = 10.0 *. atan 5.0 in (* 14 *)
let e = 10.0 *. atan -5.0 in (* -14 *)
print_int (int_of_float a);
print_int (int_of_float b);
print_int (int_of_float c);
print_int (int_of_float d);
print_int (int_of_float e)

File: simulator/atan.py
import numpy as np

def atan(x):
	return x - x**3 / 3 + x**5 / 5 - x**7 / 7 + x**9 / 9 - x**11 / 11 + x**13 / 13

if __name__ == "__main__":
	print(10 * atan(0.3))
